server/api/socketService.js

```diff
@@ -44,7 +44,7 @@ const socketService = (socket) => {
       socket.room = newGame.roomId;
       socket.join(newGame.roomId);
       logger('Created new room');
-      socket.emit(CONNECTED, roomId);
+      global.io.to(roomId).emit(CONNECTED, roomId);
```

Comment:
Why does this need to be emitted to the whole room? Why not using just `socket.emit(CONNECTED, roomId);`